app/views/layouts/card_wide.html.slim

```diff
@@ -1,3 +1,3 @@
-- card_cls 'card-wide'
+- card_cls 'card-wide sm-px2 px0'
```

Comment:
ugh sorry, actually yeah `px0` is unneeded since we are setting `padding` on the `card-wide` class (which is what we want to do since that will override the `px2` that is set previously).